mod/missions/languages/en.php

```diff
@@ -754,7 +754,7 @@
         "missions:average_number_of_applicants" => "Average number of applicants",
 
     "missions:error:department_restricted"  =>  "This opportunity is only available within: %s.<br /> Your department is currently set to: %s.<br /> If this is not correct please update your profile.",
-    "missions:groupandlevel"=>"Group/Classification",
+    "missions:groupandlevel"=>"Group/level",
```

Comment:
Wouldn't it be `Group/Classification and level`?
EX: "Group or Classification and Level", rather than "Group or Level"

Reply:
Yeah, I asked @LemieuxGen about that (and the French version). It's more of a placeholder / flagging it to be changed since the old version doesn't even mention level.